hooks/pre-command

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Create a temporary directory with both BSD and GNU mktemp
+TMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'buildkite')
+
+if [[ -z "${BUILDKITE_PLUGIN_GCP_WORKLOAD_IDENTITY_FEDERATION_AUDIENCE:-}" ]]; then
+  echo "🚨 Missing 'audience' plugin configuration"
+  exit 1
+fi
+
+if [[ -z "${BUILDKITE_PLUGIN_GCP_WORKLOAD_IDENTITY_FEDERATION_SERVICE_ACCOUNT:-}" ]]; then
+  echo "🚨 Missing 'service-account' plugin configuration"
+  exit 1
+fi
+
+echo "--- :buildkite: Requesting OIDC token from Buildkite"
```

Comment:
For symmetry with other plugins that use the muted output

https://buildkite.com/docs/pipelines/managing-log-output

```suggestion
echo "~~~ :buildkite: Requesting OIDC token from Buildkite"
```